package.json

```diff
@@ -24,16 +23,14 @@
     "search",
     "18F"
   ],
-  "browserify": {
-    "transform": [
-      "debowerify",
-      "browserify-shim"
-    ]
-  },
-  "browserify-shim": {
-    "bower_components/angular/angular.js": "angular"
-  },
   "author": "Mike Bland <michael.bland@gsa.gov> (https://18f.gsa.gov/)",
   "license": "CC0-1.0",
-  "readmeFilename": "README.md"
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "angular": "1.3.x",
+    "lunr": "^0.5.9",
+    "jquery": "^2.1.4"
```

Comment:
We _could_, but I wanted to keep the source lean. But if you think it's important to lock the versions down in the repo, we could do that.